<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTerrain</class>
 <widget class="QDialog" name="NewTerrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建地形</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>网格</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>每个网格边长:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>地图格数:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>NewTerrain</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>NewTerrain</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
